src/components/NetworkSelector/useNetworkName.ts

```diff
@@ -19,7 +19,7 @@ const fetcher = async (
 
 export const useNetworkName = (): UseNetworkNameResult => {
   const { lotusApiUrl } = useEnvironment()
-  const { data, error } = useSWR<string, Error>(
+  const { data, error } = useSWRImmutable<string, Error>(
```

Comment:
Here i think we do actually want to use `useSWR` because we want to know if we disconnect from the node 

i'll fix